spotify-track-search.el

```diff
@@ -28,8 +28,8 @@
     map)
   "Local keymap for `spotify-track-search-mode' buffers.")
 
-;; Enables the `spotify-remote-mode' in the track search buffer
-(add-hook 'spotify-track-search-mode-hook 'spotify-remote-mode)
+;; Enables the `global-spotify-remote-mode' in the track search buffer
+(add-hook 'spotify-track-search-mode-hook 'global-spotify-remote-mode)
```

Comment:
Do you think it makes sense to continue enabling this mode, given that now it is only available as a global mode?

We can agree to remove this hook and let the user decide whether he/she wants the remote mode.

Reply:
I guess I'd throw that one back to you :-) Is only allowing global mode the right approach? I wouldn't probably fire up any other minor mode. But you probably have a much better finger on the pulse of people using this.

Reply:
@danielfm I went ahead and removed this. I agree it's unnecessary